scrapper.py

```diff
@@ -169,12 +166,7 @@ def validate_config(crawler_path):
     crawler.find_articles()
 
     ID = 0
-    for article_url_test in crawler.urls:
-        ID += 1
-        article_parser = HTMLParser(article_url=article_url_test, article_id=ID)
-        try:
-            article = article_parser.parse()
-            article.save_raw()
-        except:
-            print(article_parser.article.url)
-
+    for ID, article_url_test in enumerate(crawler.urls):
```

Comment:
in python, we usually use capital letters to denote constants 
you are changing the value of `ID` after each iteration, so it is not exactly constant 
I would recommend to use `enumerate()` for defining a variable that would serve as an ascending index